egov-user-event/src/test/resources/application.properties

```diff
@@ -5,17 +5,19 @@ spring.datasource.username=postgres
 spring.datasource.password=postgres
 
 #----------------------------- FLYWAY CONFIGURATIONS ------------------------------#
-flyway.url=jdbc:postgresql://localhost:5432/hr_employee_db
-flyway.user=postgres
-flyway.password=postgres
-flyway.table=hr_employee_schema_version
-flyway.baseline-on-migrate=true
-flyway.outOfOrder=true
-flyway.locations=db/migration/main,db/migration/seed
+spring.flyway.url=jdbc:postgresql://localhost:5432/hr_employee_db
```

Comment:
This still has hrms stuff?

Reply:
Its updated in this commit  - https://github.com/egovernments/municipal-services/pull/263/commits/c3bcbee99a0cd8a58c5a4d7f21b6b391448bb47f